doc/new-book/routing.adoc

```diff
@@ -213,11 +213,30 @@ address {
 }
 ----
 
-`prefix`:: The address prefix. All messages that match this prefix will be distributed according to the distribution pattern you specify.
+`prefix` | `pattern`:: The address or group of addresses to which the address settings should be applied. You can specify a prefix to match an exact address or segment of an address. Alternatively, you can specify a pattern to match an address using wildcards.
```

Comment:
When describing the 'prefix' match I would avoid using the unqualified term 'segment' in favor of something like 'beginning segment'.   In other words:

'match an exact address or **beginning segment** of an address'

I'd apply that to all the other prefix descriptions throughout the doc.